src/python/CRABClient/JobType/Analysis.py

```diff
@@ -158,19 +158,12 @@ def run(self, filecacheurl = None):
                         ISBSize = int(re_match.group(1))
                         ISBSizeLimit = int(re_match.group(2))
                         reason  = "%sError%s:" % (colors.RED, colors.NORMAL)
-                        reason += " Input sanbox size is ~%sMB. This is bigger than the maximum allowed size of %sMB." % (ISBSize/1024/1024, ISBSizeLimit/1024/1024)
-                        ISBContent = sorted(tb.content, reverse=True)
-                        biggestFileSize = ISBContent[0][0]
-                        ndigits = int(math.ceil(math.log(biggestFileSize+1, 10)))
```

Comment:
Sorry @belforte, one other minor thing, could you remove the unused `math` import?